Dockerfile

```diff
@@ -1,8 +1,28 @@
-FROM ruby:2.5
+FROM ruby:2.5.5
```

Comment:
If we are specifying a patch version should it be the latest (2.5.6) to include all the patches?